src/main/resources/application.yml

```diff
@@ -10,7 +10,8 @@ familie:
     mottak:
       uri: https://api-gw.oera.no/familie-ef-mottak/api
   dokument:
-    uri: https://www.nav.no/familie/alene-med-barn/mellomlagring/api
+    uri: https://familie-dokument.prod-gcp.nais.io/api
```

Comment:
Hvorfor har vi to ingresser? Hva er fordelen med å gå mot nais.io vs nav.no ?

Reply:
sbs = midlertidig?

Reply:
Vi har to ingresser mot gcp: `prod-gcp.nais.io` og `nav.no`

Reply:
Ah. *flau*